internal/cmd/shell.go

```diff
@@ -196,9 +203,10 @@ func heartbeat(ctx context.Context, c *websocket.Conn, interval time.Duration) {
 		case <-ctx.Done():
 			return
 		case <-ticker.C:
-			err := c.Ping(ctx)
-			if err != nil {
-				flog.Fatal("\nFailed to ping websocket: %v, exiting...", err)
+			if err := conn.Ping(ctx); err != nil {
+				// TODO: Don't fatal, it would skip the TTY state restore and result in a broken terminal.
+				// NOTE: Prefix with \r\n to attempt to have clearer output as we might still be in raw mode.
```

Comment:
can the prefix be a constant that we can reuse for these kinds of situations?

Reply:
This is a one-off that should be avoided, as we should control how/what we output. Promoting this to a const would imply it is supported and might encourage it to be used and for just two chars, especially CRLF, I am not sure it would add value.

Reply:
I think putting "LineFeed" as constant gives it a name which makes the comment less required?

I feel like adding `\r\n` to the ends of something is never just a one-off, but I'm far less familiar with this code base.